source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,242 @@
+.. _charp-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collections using the ``Find()`` method.
```

Comment:
Suggestion: List the name of the product you are documenting at the beginning of the article so people can see quickly if they are on the wrong page.

Issue: Focus on value proposition of page in the overview. Introduce technical details like method names later in the page.

```suggestion
In this guide, you can learn how to retrieve data from MongoDB
with the MongoDB Dotnet Driver.
```